File: NETMouse - .NET release/Examples/C#/Arrays_10.cs
using ABCNET.Extensions;
using ABCNET.Utils;
using System.Linq;

namespace TestProgram
{
    internal class Program
    {
        private static void Main(string[] args)
        {
            Arr.ReadInteger(Base.ReadInteger("N:"), "Элемент {0}-ый:").Where(x => x == 0).Count().Println();
        }
    }
}

File: NETMouse - .NET release/Examples/PascalABC.NET/Arrays_10.pas
{$reference ABCNET.dll}
uses ABCNET.Utils;
uses ABCNET.Extensions;
begin
  Arr.ReadInteger(BaseU.ReadInteger('N:'), 'Элемент {0}-ый:').Where(x -> x = 0).Count().Println();
  BaseU.ReadChar();
end.